### DISCUSSION TITLE
为什么我的绳网登录不了啊 ​发帖都能正常发 ​可就是一直显示的未登录

### DISCUSSION THREAD

**Discussion by Bailinghai** [share121/inter-knot#5144] [其他]
![绳网 2024_9_18 9_46_29](https://github.com/user-attachments/assets/997048ea-d44f-4feb-9eb5-f9ac00b14cae)
![New Discussion · share121_inter-knot · Category selection 和另外 2 个页面 - 用户配置 1 - Microsoft​ Edge Beta 2024_9_18 9_46_49](https://github.com/user-attachments/assets/879b0422-2f80-4772-bd85-30ff9bc65805)

**Comment by yzhfat** [comment]
到“我的”那里选择用用户接口登录

**Reply by Bailinghai** [reply]
没用![绳网 2024_9_18 9_59_22](https://github.com/user-attachments/assets/e2e933dc-eed8-4334-a2dc-3c917112f1f2)

**Comment by yzhfat** [comment]
退后台，重进一下试试

**Reply by Bailinghai** [reply]
登上了！登上了！谢谢大佬！！！

**Comment by Bailinghai** [comment]
改回公共又变未登录了
